https://gitlab.com/eivnaes/sstp-client/-/merge_requests/2

From 3cae209750549b17c547c3025859c0468cd5fcf6 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Tue, 10 May 2022 18:04:20 +0000
Subject: [PATCH] configure.ac: fix bashism

configure scripts are expected to work with POSIX compliant shells providing
/bin/sh so use = instead of == to work with both Bash and other shells.

Signed-off-by: Sam James <sam@gentoo.org>
--- a/configure.ac
+++ b/configure.ac
@@ -129,7 +129,7 @@ PKG_CHECK_EXISTS([pppd],
 AM_CONDITIONAL(WITH_PPP_PLUGIN, test "${enable_ppp_plugin}" = "yes")
 AC_ARG_WITH([pppd-auth-notify-support],
     AS_HELP_STRING([--with-pppd-auth-notify-support], [is the auth-notifier supported in this pppd version]))
-AS_IF([test "x$with_pppd_auth_notify_support" == "xyes" ],
+AS_IF([test "x$with_pppd_auth_notify_support" = "xyes" ],
     [AC_DEFINE(USE_PPPD_AUTH_HOOK,1,[Define if pppd has support for client side authentication complete notification])
      PPPD_AUTH_NOTIFIER="yes"], 
     [PPPD_AUTH_NOTIFIER="no"])
GitLab
